app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job so far 👏🏻  However according to the project requirements, the application should ask for confirmation of email. Let's enabled the user `:confirmable` in the user model. You should see something similar to what is below:
    ```ruby
    	devise :database_authenticatable, :registerable,
    		    :recoverable, :rememberable, :validatable, :confirmable
    ````

- [ ]  Kindly, let's make sure the user can reset the password in the application. 

   You can check to see if `:recoverable` is enabled in the user model. You should see something similar to what is below:

   ```ruby
   	devise :database_authenticatable, :registerable,
   		    :recoverable, :rememberable, :validatable, :confirmable
   ```

Reply:
Thank you so much @BenMukebo , let me fix that real quick :+1:  :rocket: